### ISSUE TITLE
Typo in install instructions?

### ISSUE THREAD

**Issue by Nothinel** [xraypy/xraylarch#512]
https://github.com/xraypy/xraylarch/blob/ae6ed7961ecf84a16bd5611fb9c5cc1cc5cecde6/doc/installation.rst?plain=1#L251

Is it possible that the above mentioned line is missing "conda-forge" after the -c flag?
```
mamba install -y -c conda-forge scikit-image scikit-learn pycifrw plotly fabio pyfai
```
works while 
```
mamba install -y -c scikit-image scikit-learn pycifrw plotly fabio pyfai
```
fails.

**Comment by newville** [comment]
@Nothinel thanks!

**Comment by maurov** [comment]
Fixed by 818f128e2ab8fe8e3feae7aa0d2eccc722fd2d1c (@newville thanks!)

By the way, the current Conda package for Larch works very well, so one could simply do `mamba install -yc conda-forge xraylarch` instead of installing first the dependencies.
